onnxruntime/contrib_ops/cpu/bert/attention_parameters.h

```diff
@@ -86,7 +86,7 @@ struct GroupQueryAttentionParameters : AttentionParameters {
   int kv_hidden_size;           // hidden size of key or value
   int seqlen_past_kv_cache;     // sequence length of past kv tensor
   int seqlen_present_kv_cache;  // sequence length of present kv tensor
-  int local_window_size;        // The window size excludes current token. It only includes tokens on the left side.
+  int local_window_size;        // The window size includes new token. It only includes tokens on the left side.
```

Comment:
Update the comment to avoid confusion for "only part".
Also, in prompt, there is no "new" token.